### PR TITLE
RPM dev guide tweaks re. sha1sum and osg-build

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: docs/software/rpm-development-guide.md [hunk 1]
```diff
@@ -159,14 +164,20 @@ without the prefix component, followed by the sha1sum of the file:
 
 > `<PACKAGE>/<VERSION>/<FILE> sha1sum=<SHA1SUM>`
 
+Obtain the sha1sum by running the `sha1sum` command with the source file as an argument, i.e.
+```console
+$ sha1sum /p/vdt/public/html/upstream/<PACKAGE>/<VERSION>/<FILE>
+```
+
+!!! note
+    This feature requires OSG-Build 1.14.1 or later.
```
Comment (edquist) [inline docs/software/rpm-development-guide.md]: 1.14.0

File: docs/software/rpm-development-guide.md [hunk 2]
```diff
@@ -159,14 +164,20 @@ without the prefix component, followed by the sha1sum of the file:
 
 > `<PACKAGE>/<VERSION>/<FILE> sha1sum=<SHA1SUM>`
 
+Obtain the sha1sum by running the `sha1sum` command with the source file as an argument, i.e.
+```console
+$ sha1sum /p/vdt/public/html/upstream/<PACKAGE>/<VERSION>/<FILE>
```
Comment (edquist) [inline docs/software/rpm-development-guide.md]: You are giving the instructions for this twice?

Comment (matyasselmeci) [reply docs/software/rpm-development-guide.md]: Yeah, I feel like it's kinda hidden so I want to make sure people get it.

File: docs/software/rpm-development-guide.md (hunk 1 above)
Comment (matyasselmeci) [reply docs/software/rpm-development-guide.md]: fixed.

File: docs/software/rpm-development-guide.md (hunk 2 above)
Comment (edquist) [reply docs/software/rpm-development-guide.md]: Arright, well, for the record i feel like it's overkill, as well as giving the full path to the file in the example, but whatever you like.

**Pull request merged**